mlir-tensorrt/compiler/lib/Dialect/Plan/Transforms/CreateClosedRegions.cpp

```diff
@@ -503,11 +430,162 @@ static LogicalResult createClosedGroupOp(RewriterBase &rewriter,
       return failure();
     inputAttrs.push_back(boundsAttr);
   }
-  closedGroupOp.setInputAttrsAttr(inputAttrs);
 
+  return inputAttrs;
+}
+
+static LogicalResult materializeDestinationOperands(
+    RewriterBase &rewriter, plan::InlineGroupOp op, DataFlowSolver &solver,
+    SmallVector<DestinationOperandMaterializationResult> &destinationOperands) {
+  destinationOperands.reserve(op.getNumResults());
+  for (OpResult res : op->getOpResults()) {
+    FailureOr<DestinationOperandMaterializationResult> destResult =
+        materializeDestinationOperand(rewriter, res.getLoc(), op,
+                                      res.getResultNumber(), solver);
+    if (failed(destResult)) {
```

Comment:
Nit: remove curly braces for single-statement bodies of `if` statements 
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements